styles/gallery.css

```diff
@@ -0,0 +1,322 @@
+
+ul {
+    list-style-type: none;
+    margin:0px;
+    padding:0;
+    overflow: hidden;
+    background-color:#4ABDAC; 
+}
+
+li {
+    float: left;
+    border-right: 1px solid #EEE;
+}
+
+li a {
+    display:block;
+    /* color:#4ABDAC; */
+    text-align:center;
+    padding: 14px 16px;
+    text-decoration:none;
+}
+
+/* Change the link color to #f79833dc on hover */
+li a:hover{
+    background-color:#f79833dc;
+}
+
+body {
+    background-color:#EEE;
+    width: 600px;
+    padding: 10rem;
+}
+/* .cat-container {
```

Comment:
It looks like you had an extremely frustrating time with the `grid`. 😞  I empathize deeply with that. It took me a lot of time to wrap my head around what grid was even trying to do and how to avoid all of the cumbersome `grid-column-start` and `grid-column-end` (etc.) rules. Good news: you can! Try out what happens if you remove your `row` and `col` divs and put back your `cat-container`-class `section`, using these CSS rules:

```css
.cat-container {
    background-color:#4ABDAC;
    display: grid;
    grid-template-columns: repeat(3, 1fr);
    padding: 20px;
    gap: 20px;
}

img {
    max-width: 100%;
    padding: unset;
}

body {
    width: unset;
}
```

You might need to re-order the cat `img`s, but you get the grid it looks like you wanted with a lot less hassle.

I appreciate that you fought hard with this, and I hope that the experience helps you calibrate your sense of what level of frustration you ought to reach before looking for advice from more experienced devs. That will be really valuable in industry!